auth.h

```diff
@@ -42,7 +42,7 @@ gboolean janus_auth_check_signature(const char *token, const char *realm);
  * @param[in] token The token to validate
  * @param[in] realm The token realm
  * @param[in] desc The descriptor to search for
- * @returns TRUE if the token is valid, not expired and contains the descriptor, FALSE otherwise */
+ * @returns When the token based authentication enabled: TRUE if  the token is valid, not expired and contains the descriptor, FALSE otherwise. When the token based authentication disabled: always TRUE. */
```

Comment:
I'd say "is enabled", rather than just "enabled", in both additions.

Reply:
fixed